src/modules/upcoming-events.module/fields.json

```diff
@@ -1 +1,12 @@
-[]
+[
+  {
+    "label": "Page title",
```

Comment:
Since the module will likely be used in a secondary fashion on a page, I think a different label/name might make sense. Maybe "Heading"?